.github/workflows/ci.yml

```diff
@@ -41,7 +41,7 @@ jobs:
 
       - name: Find the Go Build Cache
         id: go
-        run: echo "::set-output name=cache::$(make go.cachedir)"
+        run: echo "::set-output name=cache::$(go env GOCACHE)"
```

Comment:
@turkenh just checking -- I believe we avoid the situation described in https://github.com/crossplane/crossplane/blob/c0de80c1ea2102b9569754d78bea292c46e62927/Makefile#L162 because we aren't including https://github.com/upbound/build/blob/bd63a4167ae20a71788537217b022fced8f2f854/makelib/helm.mk#L48. Is that your understanding as well?

Reply:
Yes, I think so. 
Do you have any ideas on how to fix this in the build submodule?

Reply:
@turkenh from what I could tell from docs we could still use `HELM_HOME` https://helm.sh/docs/faq/changes_since_helm2/#xdg-base-directory-support

Reply:
I remember `HELM_HOME` didn't work at that time and I've found this solution by reading the same documentation page :)
However, I don't remember the `HELM_PATH_*` variables which might be introduced later. Added to my todo list, I'll give it a try once I have time.

Reply:
And I believe we can remove HELM 2 support from build submodule which I would make this work a bit more attractive 😅